### PR TITLE
add support for interface based rules

### PR DESCRIPTION
**Pull request by mzanetti**

### CODE REVIEW

**Comment by mzanetti** [comment]
Jenkins, retest this please

**Comment by mzanetti** [comment]
Jenkins, retest this please

**Comment by mzanetti** [comment]
Jenkins, retest this please

**Comment by mzanetti** [comment]
Jenkins, retest this please

File: tests/auto/restrules/testrestrules.cpp [hunk 1]
```diff
@@ -540,6 +540,7 @@ void TestRestRules::addRemoveRules()
     QNetworkRequest request(QUrl("https://localhost:3333/api/v1/rules"));
     QVariant response = getAndWait(request);
     QVariantList rulesList = response.toList();
+    qDebug() << "**** rukes" << rulesList;
```
Comment (t-mon) [inline tests/auto/restrules/testrestrules.cpp]: rules

Comment (mzanetti) [reply tests/auto/restrules/testrestrules.cpp]: fixed

**Comment by mzanetti** [comment]
Jenkins, land this please

**Comment by mzanetti** [comment]
merged

**Pull request closed without merging**